src/bitops.c

```diff
@@ -765,7 +766,7 @@ void bitopCommand(client *c) {
     zfree(objects);
 
     /* Store the computed value into the target key */
-    if (maxlen) {
+    if (maxlen || nullobjs < numkeys) {
```

Comment:
maxlen implies that "nullbobjs < numkeys", since there must be at least one non-null object. I would either:
* Drop the maxlen check here.
* Move the delete code up to the top. Since really the code should be, if all keys are null, delete the dest key.

Reply:
First part fixed.
But if we move the delete code up to the top, we have to clean twice?  Or we use goto to clean?